### PR TITLE
Reduced height of search option

### PR DESCRIPTION
**Pull request by Nikhil743**
#867

## Changes proposed
Reduced height of search option

## Screenshots
search box

### CODE REVIEW

**Comment by Nikhil743** [comment]
Reduced height of search option

**Review by Nikhil743 — COMMENTED** [review]
c02baef Reduced height of search option

**Comment by Nikhil743** [comment]
29b5a69 
[Resize 'home page' and make it responsive]

**Comment by codervivek5** [comment]
solve conflict plz

**Comment by Nikhil743** [comment]
c02baef [Reduced height of search option]

**Comment by codervivek5** [comment]
kindly solve conflict

**Comment by Nikhil743** [comment]
solved

**Review by Haris-Mohanty — COMMENTED** [review]
@Nikhil743, Solve the conflict first.

**Comment by Nikhil743** [comment]
I have solve this conflicts @codervivek5 @Haris-Mohanty

**Comment by codervivek5** [comment]
due to the multiples PR its occurred again plz solve it then I will merge it

**Comment by Nikhil743** [comment]
I have solve this conflicts @codervivek5

File: style.css [hunk 1]
```diff
@@ -392,9 +396,15 @@ body.dark {
 
 .Accessories-box {
 	border-radius: 14px;
+
 	width: 125px;
 	height: 220px;
 	margin: 0 4em;
+=======
```
Comment (codervivek5) [inline style.css:403]: why this equal ????

File: style.css [hunk 2]
```diff
@@ -380,9 +380,13 @@ body.dark {
 
 /* product container */
 
-.product-container {
-	height: 25vh;
+
+
 	width: 100vw;
+=======
```
Comment (codervivek5) [inline style.css:386]: why use equal?????

**Comment by codervivek5** [comment]
kindly solve conflict

**Pull request closed without merging**